### ISSUE TITLE
Create Content for the Design System Accessibility Page 

### ISSUE THREAD

**Issue by MikeWalrath** [grommet/hpe-design-system#1039]
https://design-system.hpe.design/foundation/accessibility

This issue aims to effectively communicate/educate the importance of accessibility standards and how the Design System is helping create consistent accessible UX to HPE. 
- Copy and imagery needed
- Supporting documentation
- Related articles showing the importance of accessible applications

[Shimi]:
- Presenting different tool options that can help guide and measure the quality of accessibility
- Point to  configuration guidelines for screen readers
- Listing contact people or resources that can help evaluate the accessibility of a webpage
- Collect info from VPAT discussions, and be transparent about the results 

It would be great to post initial thoughts and drafts within this issue for review. We can then collaborate on the design and implementation of the website.

**Comment by cherih** [comment]
HPE Digital Guidelines linked to from Brand Central:

Accessibility
http://digitalguidelines.hpecorp.net/standards/accessibility/overview
http://digitalguidelines.hpecorp.net/standards/accessibility/accessibilityathpe

**Comment by cherih** [comment]
From Edgar’s GLC NFR doc:

Accessibility 
1.	WCAG 2.1 AA compliance is the accessibility standard we are aiming to achieve.  See https://www.w3.org/TR/WCAG21/. This includes things like: 
a.	Correct use of color. 
i.	Not using color as the sole means of conveying information. 
ii.	Ensuring sufficient contrast. 
b.	Keyboard navigation. 
c.	Screen reader support. 
d.	…and more.  Note that Grommet and the HPE theme have a minimum AA compliance requirement, so that’s a good starting point.

**Comment by vavalos5** [comment]
@cherih I found this article yesterday and thought it was helpful. Thought I'd share. :)

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility

**Comment by cherih** [comment]
@vavalos5 - thank you for sharing the link to this article! After a quick skim through it, this looks like a great external article to reference.

**Comment by ericsoderberghp** [comment]
We should be including guidance for how to test and evaluate accessibility.

**Comment by ShimiSun** [comment]
@amandadupell should be able to do the first pass of reviewing and elaborating on missing content after her recent accessibility research and exploration of tools and overall guidance she had been exploring for the last few weeks.

**Comment by taysea** [comment]
Closing, the accessibility page now has a variety of topics and content from Amanda's work.
